### PR TITLE
Complete User stories 11-12: user creation sad path messaging

### LINKED ISSUE #11: User Story 11, User Registration Missing Details
As a visitor
When I visit the user registration page
And I do not fill in this form completely,
I am returned to the registration page
And I see a flash message indicating that I am missing required fields

### PR DESCRIPTION
**Pull request by DanielEFrampton**
## Description
Resolves #11, #12 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## RSpec Results
```
Finished in 3.35 seconds (files took 4.19 seconds to load)
107 examples, 0 failures

Coverage report generated for RSpec to /Users/dframpton/turing/2module/group_projects/monster_shop_p

art_1/coverage. 1311 / 1311 LOC (100.0%) covered.
```

### CODE REVIEW

**Comment by DanielEFrampton** [comment]
There's some janky code in here, so please leave questions about what we did and why; we're up for refactoring later.

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -1,21 +1,21 @@
 <h2>New User Registration</h2>
 
-<%= form_tag "/users" do %>
+<%= form_tag "/register" do %>
```
Comment (mel-rob) [inline app/views/users/new.html.erb:3]: We might want to think about using form_with here vs. form_tag. That way we don't have to include all those conditionals to pre-populate the form.

**Review by mel-rob — APPROVED** [review]

File: app/views/users/new.html.erb (hunk 1 above)
Comment (philjdelong) [reply app/views/users/new.html.erb:3]: Cool, I haven't seen form_with yet

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -3,15 +3,19 @@ def new
   end
 
   def create
-    user = User.new(user_params)
+    @user = User.new(user_params)
```
Comment (philjdelong) [inline app/controllers/users_controller.rb:6]: Makes sense here.. we're potentially rendering a view with instances needed.

**Review by philjdelong — APPROVED** [review]

**Pull request merged**